cmd/kubeadm/test/util.go

```diff
@@ -45,9 +45,9 @@ func SetupTempDir(t *testing.T) string {
 	return tmpdir
 }
 
-// SetupInitConfigurationFile is a utility function for kubeadm testing that writes a master configuration file
+// SetupInitConfigurationFile is a utility function for kubeadm testing that writes a control-plane configuration file
```

Comment:
@xichengliudui please change:
`master configuration file`
to `InitConfiguration file`

[1]

Reply:
thanks, has been updated.